website/blog/2025-02-04-replication-ferretdb-enabling-high-availability-performance.md

```diff
@@ -0,0 +1,114 @@
+---
+slug: replication-ferretdb-enabling-high-availability-performance
+title: 'Replication in FerretDB: Enabling High Availability and Performance'
+authors: [alex]
+description: >
+  Learn how replication in FerretDB can help you create a reliable and fault-tolerant environment for your applications.
+image: /img/blog/replication.jpg
+tags: [community, cloud, open source]
+---
+
+![Replication in FerretDB](/img/blog/replication.jpg)
+
+Speed and reliability are critical for all databases – every millisecond matters.
+When serving data to your users, every second of downtime can lead to lost revenue and unhappy customers.
+
+<!--truncate-->
+
+For applications that demand high availability and disaster recovery, replication is an essential feature.
+Now, with the release of FerretDB v2 (which uses PostgreSQL with DocumentDB extension as the storage engine) and its replication capabilities, developers and businesses can create fault-tolerant and highly available database environments.
+
+In this blog post, we'll discuss replication, its importance and benefits, and how FerretDB can help you create a reliable and fault-tolerant environment.
+
+## What is replication?
+
+Replication is a database technique for storing and distributing multiple copies of data across different servers.
+It works by maintaining one primary database where all writes happen and one or more read-only replicas that sync with the primary.
```

Comment:
This is how WAL works, but isn't it possible to have multiple primaries if synchronous replication is used?

Reply:
We only support WAL streaming for now.

https://docs.ferretdb.io/guides/replication/#primary-server